ivy/functional/frontends/tensorflow/math.py

```diff
@@ -433,6 +433,11 @@ def minimum(x, y, name=None):
     return ivy.minimum(x, y)
 
 
+@to_ivy_arrays_and_back
+def mod(x, y, name=None):
+    return ivy.remainder(x, y)
```

Comment:
I've omitted `check_tensorflow_casting` similarly to the `floormod` function. Is this as expected?

Reply:
Tests do pass, but it may be better practice to just add it in!

Reply:
I've it added! Thank you :)